source/Nuke.Common/Tooling/ToolPathResolver.cs

```diff
@@ -169,6 +170,9 @@ public static string GetPathExecutable(string pathExecutable)
                 ? @"C:\Windows\System32\where.exe"
                 : "/usr/bin/which";
 
+            if (!File.Exists(locateExecutable))
```

Comment:
`where.exe` doesn't exist on a bunch of docker containers (well, on ` mcr.microsoft.com/dotnet/sdk:5.0` at least).